src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryService.scala

```diff
@@ -133,6 +149,18 @@ class LibraryService (protected val argUserInfo: UserInfo,
     }
   }
 
+  def getAttributes(ns: String, name: String): Future[PerRequestMessage] = {
```

Comment:
Yeah, I agree that the asymmetry here is a bit ugly. I didn't change the response of `updateAttributes` (now `updateLibraryMetadata`) because that could break compatibility for anyone using it. But at the same time, it makes no sense for the put-metadata endpoint to accept an entire workspace object.

I did rename the case classes and methods to try to keep code understandable.

"metadata" is short for "Library metadata" which is anything in the library namespace. The put-metadata endpoint enforces that at line 145.